.github/workflows/built_and_test.yml

```diff
@@ -544,8 +544,8 @@ jobs:
         if: always()
         continue-on-error: true
         run: |
-          ls -lahr test/mwlog test/mwlog/*
-          tail -n +1 test/mwlog/*/*.log
+          ls -lahr test/log test/log/*
+          tail -n +1 test/log/*/*.log
```

Comment:
Might be a good chance to change this file's name :)

Reply:
Not sure what you mean in terms of the file's name? 

_However line 548 here maybe should be changed to `tail` only `test/log/*/wikibase/*.log` to have the same behaviour as before. Wherever this log tailing appears this currently will also be tailing the logs Docker log and setup log... Not necessarily wrong, but that is a lot of logging and different than before. Hmm... I am ok leaving this, but_

Reply:
I meant `built` -> `build`.

As for logging, if it's not too much clutter, I don't see any harm in also having the non-wikibase logs.

Reply:
Are you still planning to rename the file? The conversation was already resolved, but as I see it states still "built" in the filename.